pkg/controller/kibana/init_configuration.go

```diff
@@ -0,0 +1,65 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package kibana
+
+import (
+	kbv1 "github.com/elastic/cloud-on-k8s/pkg/apis/kibana/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+const (
+	InitConfigContainerName = "elastic-internal-init-config"
+
+	// InitConfigScript is a small bash script to prepare the Kibana configuration directory
+	InitConfigScript = `#!/usr/bin/env bash
+set -eux
+
+init_config_initialized_flag=/usr/share/kibana/config/elastic-internal-init-config.ok
+
+if [[ -f "${init_config_initialized_flag}" ]]; then
+    echo "Kibana configuration already initialized."
+	exit 0
+fi
+
+echo "Setup Kibana configuration"
+
+ln -sf /mnt/elastic-internal/kibana-config/* /mnt/elastic-internal/kibana-config-local/
```

Comment:
this should probably reuse 2 constants instead of being hardcoded?

Reply:
Initially I though it was less readable, but let's avoid duplication.